extensions/libraries/redcore/controller/admin/base.php

```diff
@@ -242,25 +270,30 @@ public function reorder()
 	 * Method to save the submitted ordering values for records.
 	 *
 	 * @return  boolean  True on success
+	 * @since   1.0.0
+	 * @throws  \Exception
 	 */
 	public function saveorder()
 	{
-		// Check for request forgeries.
-		JSession::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
+		$cid = $this->validateRequestCids();
+
+		if (!$cid)
+		{
+			return false;
+		}
 
-		// Get the input
-		$pks = $this->input->post->get('cid', array(), 'array');
 		$order = $this->input->post->get('order', array(), 'array');
 
 		// Sanitize the input
-		ArrayHelper::toInteger($pks);
```

Comment:
Here

Reply:
This has been there for quite some time and Joomla changed this many versions ago.  I'll create a separate issue to handle this.